sites/next.skeleton.dev/src/content/docs/tutorials/figma/chapter-1.mdx

```diff
@@ -0,0 +1,110 @@
+---
+layout: '@layouts/LayoutDoc.astro'
+title: Chapter 1 - Setup & Usage
+description: Learn how to set up and use the Skeleton Figma Design System.
+order: 0
+---
+
+export const components = componentSet;
+
+import Process from '@components/docs/Process.astro';
+import ProcessStep from '@components/docs/ProcessStep.astro';
+import NavGrid from '@components/docs/NavGrid.astro';
+
+import { Image } from 'astro:assets';
+import SvgSkeleton from '@images/tutorials/figma/skeletonFileIcon.svg';
+import ImgCreateNewProject from '@images/tutorials/figma/chapter-1/01_createNewProject.webp';
+import ImgImportProject from '@images/tutorials/figma/chapter-1/02_importProject.webp';
+import ImgPublish from '@images/tutorials/figma/chapter-1/03_publish.webp';
+import ImgCreateNewFile from '@images/tutorials/figma/chapter-1/04_createNewFile.webp';
+import ImgSelectLibrary from '@images/tutorials/figma/chapter-1/05_selectLibrary.webp';
+import ImgUseLibrary from '@images/tutorials/figma/chapter-1/06_useLibrary.webp';
+
+---
+
+## How to Install the Figma Library
+
+For best results, a <a href="https://www.figma.com/pricing/#figma" class="chip preset-filled mx-1" target="_blank">Figma Professional Team Plan</a> (or higher) is recommended.
+
+<Process>
+    <ProcessStep step="1">
+        ### Purchase the Figma Asset
+
+        <div class="card-enhanced border-[1px] border-surface-200-800 space-y-2 p-4 md:p-8 flex flex-row justify-between items-center">
+            <div className="flex flex-row gap-4 items-center">
+            <Image src={SvgSkeleton} class="rounded-container" alt="Skeleton" />
+            <div className="flex flex-col gap-1">
+                <h6 className="h6">Skeleton Figma Design System</h6>
+                <p>Complete set of themed components for Skeleton v3</p>
+            </div>
+            </div>
+            <a href="https://polar.sh/skeleton-kit/" type="button" class="btn preset-filled-primary-500" target="_blank">Purchase</a>
+        </div>
+    </ProcessStep>
+
+    <ProcessStep step="2">
+        ### Create a New Project
+
+        In Figma, go to `Home`¹ and select `All projects` from the left sidebar². On the right side, click `+ Project`³ and name the project `Libraries`⁴.
```

Comment:
I feel like we're way overusing the `code` highlighting, which feels very busy and a bit harder to read. Basically: "if everything is important, nothing is". So I'd suggest trying to strip this down to simpler and more straight forward text. Maybe use italics instead?

Additionally the super/sub text should probably be markup rather than unicode characters. This isn't possible in markdown, but MDX supports HTML, which should enable:

* `<sup></sup>`
* `<sub></sub>`

---

Alternatively, I think the small numerals may get lost with certain fonts and font sizes from various themes. Maybe consider rewriting using bulleted lists:

```
Follow these steps in Figma:

1. Go to the Home icon (top-left).
2. Select the All Projects option from the left sidebar.
3. Select Project from the right-side of the page.
4. Provide your project name in the dialog prompt.
```